django/db/models/aggregates.py

```diff
@@ -53,9 +53,9 @@ def get_source_expressions(self):
             return source_expressions + [self.filter]
         return source_expressions
 
-    def set_source_expressions(self, exprs):
-        self.filter = self.filter and exprs.pop()
-        return super().set_source_expressions(exprs)
+    def set_source_expressions(self, expression):
+        self.filter = self.filter and expression.pop()
+        return super().set_source_expressions(expression)
```

Comment:
This is not a typo, please revert.

Reply:
Also this was pushed after I reviewed the changes, that's not ok in my book 😛

Reply:
I am sorry, my mistake. Changes reverted 😎